### PR TITLE
Updated descriptions Royal Knight Fighter subclass abilities

### PR DESCRIPTION
**Pull request by LunaKB**
- Changed descriptions for the following Royal Knight Fighter subclass abilities:
-- Rallying Cry
-- Inspiring Surge
- Corrected title for Inspiring Surge power. It used to say "Preserve Life". It now says "Inspiring Surge".

This should provide clarity to the players.

### CODE REVIEW

File: SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ internal MartialRoyalKnight()
             .AddToDB();
 
         var effectDescription = EffectDescriptionBuilder
-            .Create(PowerDomainLifePreserveLife.EffectDescription)
+            .Create()
```
Comment (ThyWoof) [inline SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs:34]: Why are you touching these effects?

**Review by ThyWoof — CHANGES_REQUESTED** [review]

File: SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs [hunk 2]
```diff
@@ -57,11 +57,11 @@ internal MartialRoyalKnight()
             .AddToDB();
 
         var powerRoyalKnightInspiringSurge = FeatureDefinitionPowerBuilder
-            .Create(PowerDomainLifePreserveLife, "PowerRoyalKnightInspiringSurge")
```
Comment (ThyWoof) [inline SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs:60]: Why change this power?

File: SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs [hunk 3]
```diff
@@ -57,11 +57,11 @@ internal MartialRoyalKnight()
...
+            .Create("PowerRoyalKnightInspiringSurge")
             .SetGuiPresentation(Category.Feature, SpellDefinitions.Heroism)
             .SetUsesFixed(ActivationTime.BonusAction, RechargeRate.LongRest)
             .SetEffectDescription(EffectDescriptionBuilder
-                .Create(PowerDomainLifePreserveLife.EffectDescription)
```
Comment (ThyWoof) [inline SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs:64]: Why change this effect?

**Review by ThyWoof — CHANGES_REQUESTED** [review]

File: SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs (hunk 1 above)
Comment (Pikachar2) [reply SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs:34]: The ability effectively gives Action Surge to another character. It has nothing to do with Preserve Life. Which is the Life Cleric's Channel Divinity. To the best of our knowledge it seems that it's just a description/title that's setup as kind of a template in these instances. And on top of that, it was named as Preserve Life when looking for the ability in Combat/from the Action bar.

Comment (ThyWoof) [reply SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs:34]: These have nothing to do with descriptions but the template we copy behavior from. I don’t believe it should be changed. Only the translations

File: SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs (hunk 2 above)
Comment (LunaKB) [reply SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs:60]: When you select the power as a bonus action, the name for power was incorrect. Removing this makes the text correct. Here's what the UI looks like without the fix:

UnfinishedBusiness-pull2653

File: SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs (hunk 3 above)
Comment (LunaKB) [reply SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs:64]: This was changed in the effort to get the text for the power Inspiring Surge to no longer say "Preserve Life". I'll update this with this put back.

File: SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs (hunk 1 above)
Comment (LunaKB) [reply SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs:34]: This was changed in the effort to get the text for the power Inspiring Surge to no longer say "Preserve Life". I'll update this with this put back.

**Comment by LunaKB** [comment]
Updated pull request according to comments.

File: SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs (hunk 3 above)
Comment (ThyWoof) [reply SolastaUnfinishedBusiness/Subclasses/MartialRoyalKnight.cs]: this is an effect description. It's nothing to do with the description on screen.

**Comment by ThyWoof** [comment]
I confirm it’s indeed showing the wrong title even though the json dump says otherwise. Will look into it

**Comment by LunaKB** [comment]
Alright, if it will be fixed later, I'll update with the preserve life put back.

**Comment by LunaKB** [comment]
Updated pull request according to comments.

**Comment by ThyWoof** [comment]
found out what the issue was. PowerDomainLife also had a short title description set. This was causing the wrong title to display.

**Pull request merged**